packages/wdio-cli/src/interface.ts

```diff
@@ -23,12 +23,6 @@ interface CLIInterfaceEvent {
     error?: TestError
 }
 
-interface Job {
```

Comment:
I don't see a reason to move these types into `@wdio/types` as these are nowhere used besides in this file.